src/app/accounts/lock.component.html

```diff
@@ -36,5 +36,10 @@
                 {{'logOut' | i18n}}
             </button>
         </div>
+        <div class="button" *ngIf="supportsBiometric">
+            <a class="btn primary block" appBlurClick (click)="unlockBiometric()">
+                <i class="fa fa-unlock-alt" aria-hidden="true"></i> <b>{{'unlockWithWindowsHello' | i18n}}</b>
```

Comment:
Would we want this text and/or action to be OS/environment aware so that we weren't using a generic term like (unlock or supports Biometric) when we really mean, "Windows Hello" vs. "Touch ID" (macOS)?

Reply:
I was thinking on switching depending on platform. On windows it would say "Unlock with Windows Hello" while on Mac "Unlock with Touch ID". This initial draft is just a working version of Windows Hello.